pontoon/sync/core.py

```diff
@@ -28,6 +31,34 @@ def sync_project(db_project, now):
         changeset.execute()
 
 
+def serial_task(timeout, **celery_args):
+    """
+    Decorator ensures that there's only one running task with given task_name.
+    Decorated tasks are bound tasks, meaning their first argument is always their Task instance
+    :param timeout: time after which lock is released.
+    """
+    def wrapper(func):
+
+        @shared_task(bind=True, **celery_args)
+        @wraps(func)
+        def wrapped_func(self, *args, **kwargs):
+            # Moved import here to mock cache during tests.
```

Comment:
I KNEW I forgot to mention something last pass over this. This comment is inaccurate. Also, we can move this import with the rest of the imports now.